code-challenges.js

```diff
@@ -31,24 +56,62 @@ const temp2 = 350
 const temp3 = 212
 // Expected output: "212 is at boiling point"
 
+//Function
+const boilingPoint = (temp1) => {
+    if(temp1 > 212){
+        return `${temp1} is above boiling point`
+    } else if(temp1 == 212){
+        return `${temp1} is at boiling point`
+    } else {
+        return `${temp1} is below boiling point`
+    }
+
+}
+//console.log(boilingPoint(temp1))
+//output: 42 is below boiling point
+
+//console.log(boilingPoint(temp2))
+//output: 350 is above boiling point
+
+//console.log(boilingPoint(temp3))
+//output: 212 is at boiling point
+
```

Comment:
great job!  Just one note - make sure to use the strictly equals operator to avoid any side effects.  :)